### PR TITLE
Temporarily skip flutter build apk for native_assets tests.

### PR DESCRIPTION
**Pull request by matanlurey**
Work around for https://github.com/flutter/flutter/issues/158560.

### CODE REVIEW

File: packages/flutter_tools/test/integration.shard/isolated/native_assets_without_cbuild_assemble_test.dart [hunk 1]
```diff
@@ -138,8 +138,11 @@ void main(List<String> args) async {
           mode: ProcessStartMode.inheritStdio,
         );
         expect(await process.exitCode, 0);
-      });
+      },);
```
Comment (bkonyi) [inline packages/flutter_tools/test/integration.shard/isolated/native_assets_without_cbuild_assemble_test.dart:141]: Are you preparing for when the formatter is going to run here? ;)

**Review by bkonyi — APPROVED** [review]
LGTM!

File: packages/flutter_tools/test/integration.shard/isolated/native_assets_without_cbuild_assemble_test.dart (hunk 1 above)
Comment (matanlurey) [reply packages/flutter_tools/test/integration.shard/isolated/native_assets_without_cbuild_assemble_test.dart:141]: In my head the formatter is already on.

File: packages/flutter_tools/test/integration.shard/isolated/native_assets_without_cbuild_assemble_test.dart [hunk 2]
```diff
@@ -138,8 +138,11 @@ void main(List<String> args) async {
...
     },
+    // TODO(matanlurey): Debug why flutter build apk often timesout.
+    // See https://github.com/flutter/flutter/issues/158560 for details.
+    skip: buildCommand == 'apk' ? 'flutter build apk times out' : false,
```
Comment (christopherfujino) [inline packages/flutter_tools/test/integration.shard/isolated/native_assets_without_cbuild_assemble_test.dart:145]: gotta love `dynamic`

**Comment by christopherfujino** [comment]
Actually looks like autosubmit merged this with a lot of failing builds.

**Comment by matanlurey** [comment]
> Actually looks like autosubmit merged this with a lot of failing builds.

Yeah I wonder if this is a cocoon regression?

**Comment by christopherfujino** [comment]
I filed https://github.com/flutter/flutter/issues/158892 to track

**Pull request merged**